index.js

```diff
@@ -36,8 +36,11 @@ class LambdaTestHelper extends Helper{
 
   getBody(testTitle, status){
     let body = {}
+    if(status === "passed" || status === "failed") body.status_ind = status
```

Comment:
lets put a null check here and update status accordingly.

Reply:
Made the required changes